public/env/env.js.example

```diff
@@ -3,3 +3,4 @@ window.DATA_DIR_URL = 'https://path/to/data';
 window.DOCS_BASE_URL = 'https://path/to/docs';
 window.SCHEMAS_DIR = 'https://path/to/schemas';
 window.GF_ROOT_URL = 'https://grafana.example.com';
+window.QONTRACT_API_URL = 'http://localhost:5000/notifications';
```

Comment:
would it make sense to have `window.QONTRACT_API_URL` point at the root of the service and not specifically to `/notifications`? I can imagine we are going to use this in the future for things other than notifications, so we might as well make the change now.

Reply:
Yes it makes sense! Will point `window.QONTRACT_API_URL` to the root instead.